test/docs.js

```diff
@@ -189,7 +191,7 @@ describe('documentation', function() {
 
           // in:  fs.anyMethod('--any-file-path--
           // out: fs.anymethod('--a-temporary-file--
-          .replace(/(fs\.[^(]+\(['"])([^'"]*)/g, '$1' + temporaryFile.name);
```

Comment:
I know this isn't fun, but can you `console.log(temporaryFile.name)` here so that when AppVeyor runs it, we can see it?

It seems like something with the `replace` line could be messed up, since the error is:

> ENOENT, no such file or directory 'C:\UsersappveyorAppDataLocalTemp	mp-2676Xj4IRwkvTWvY.tmp'